### PR TITLE
Azure: Add support for configuring storage account type

### PR DESCRIPTION
**Pull request by discordianfish**

### CODE REVIEW

**Comment by s-urbaniak** [comment]
ok to test

**Comment by s-urbaniak** [comment]
LGTM, thanks! but leaving that one to @metral/@justaugustus for another pass.

**Comment by s-urbaniak** [comment]
@discordianfish do you mind to run `terraform fmt`? CI says:
```
misformatted files (run 'terraform fmt .' to fix): modules/azure/master/variables.tf platforms/azure/variables.tf
```

File: examples/terraform.tfvars.azure [hunk 1]
```diff
@@ -19,6 +19,9 @@ tectonic_azure_dns_resource_group = "tectonic-dns-group"
 // Instance size for the etcd node(s). Example: Standard_DS2_v2.
 tectonic_azure_etcd_vm_size = "Standard_DS2_v2"
 
+// (optional) Storage account type for the etcd node(s). Default: Premium_LRS |
+// tectonic_azure_etcd_storage_account_type = "Standard_LRS"
```
Comment (justaugustus) [inline examples/terraform.tfvars.azure]: @discordianfish The defaults are correctly set, however the example tfvars don't reflect that.
Can you run `make docs examples` and then push?

Comment (discordianfish) [inline examples/terraform.tfvars.azure]: Oh wow, nice tooling. Didn't know about that :). Fixed!

Comment (justaugustus) [inline examples/terraform.tfvars.azure]: Awesome! Thanks @discordianfish!

**Comment by justaugustus** [comment]
LGTM

**Comment by metral** [comment]
LGTM

**Comment by s-urbaniak** [comment]
all green, hence merging

**Pull request merged**